client/http_client.go

```diff
@@ -103,16 +103,33 @@ func APIErrorFrom(resp *http.Response) Error {
 		}
 	}
 
+	if apiErr.Message == "" {
+		if apiErr.Fallback != "" {
+			apiErr.Message = apiErr.Fallback
+		} else {
+			apiErr.Message = "unexpected status code"
+		}
+	}
+
 	return apiErr
 }
 
 // Error is an error returned by the API server.
 type Error struct {
-	Code    int    `json:"status_code"`
-	Message string `json:"error_message"`
+	Code     int    `json:"status_code"`
+	Message  string `json:"error_message"`
+	Fallback string `json:"message"`
```

Comment:
When does this happen? Seems like something we should remove in the api side if its inconsistent?

Reply:
Run the following command from api repo's directory, and you will see those.

```shell
grep -irn --color '"message"' .
```